OpenEdXMobile/res/layout/row_discussion_comments_comment.xml

```diff
@@ -27,6 +27,8 @@
             style="@style/discussion_regular_text"
             tools:text="This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment. This is the text of a comment." />
 
+        <!-- This container layout is only defined to add bottom padding to the parent view, when
+         the enclosed TextView's visibility is set to GONE -->
```

Comment:
Nitpicks 😜

- It would look better to have the text aligned.
- There should be a period at the end.